examples/mobilenet/model/requirements.txt

```diff
@@ -0,0 +1,3 @@
+numpy==1.16.2
+keras
```

Comment:
Fix keras version. After recent update, package `mobilenetv2` was renamed to `mobilenet_v2`. I suggest using `keras==2.2.5` and changing mobilenet example code from `mobilenetv2` to `mobilenet_v2`